### ISSUE TITLE
Newbie blogger

### ISSUE THREAD

**Issue by pgmckillop** [OpenLiveWriter/OpenLiveWriter#260]
I used WLW some time ago but got out of the habit of blogging. I want to log into a WordPress installation on my hosted account. It's not a native WordPress blog. OLW says my username and password are incorrect but i have double checked and they are correct.
Can I log into this type of blog installation using OLW?

**Comment by ScottIsAFool** [comment]
You certainly can. Just to double check, when you say it's not a native wordpress blog, you just mean it's not on wordpress.com, right? Are you able to provide a link to the blog itself?

**Comment by pgmckillop** [comment]
Hi Scott,

 

Thanks for the quick reply. You are correct. I pay for webhosting and I installed a WordPress on it; I think its version 4.4. It’s on Linux hosting and it’s at http://paulmckillop.uk.

 

Paul

**Comment by ScottIsAFool** [comment]
Hmmm, everything looks ok. When adding it, I assume you're just selecting Wordpress, then putting in your blog address along with your username/password?

**Comment by pgmckillop** [comment]
Yes,  I both WordPress and other. My user name has a space in it. Is that an issue?

**Comment by pgmckillop** [comment]
I’ve set up another account to see if that works. Same thing. I attach a screenshot.

**Comment by ScottIsAFool** [comment]
The screenshot never showed up, I think you have to add them in the website itself, not via email.

**Comment by pgmckillop** [comment]
Ok. Just tried an old wordpress.com blog account and the same thing happens there. I wonder if a re-install would fix it.

**Comment by willduff** [comment]
Can you follow the [instructions for sending your Writer log file](https://github.com/OpenLiveWriter/OpenLiveWriter/wiki/Instructions-for-sending-your-Writer-log-file) and reference issue #260 in your email? Thanks!

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
